package.json

```diff
@@ -10,6 +10,8 @@
   },
   "dependencies": {
     "lodash": "^4.17.4",
+    "nodeless-trakt": "^1.0.0",
+    "query-string": "^4.3.4",
```

Comment:
@jenriquejr As React Native doesn't really run on top of NodeJS we don't have the `querystring` core module, nor `crypto`, `http`, `events`, `stream` for that matter.